### PR TITLE
Fix to topomap colormap selection.

### PR DESCRIPTION
**Pull request by jaeilepp**
Closes https://github.com/mne-tools/mne-python/issues/3701

### CODE REVIEW

File: mne/viz/topomap.py [hunk 1]
```diff
@@ -1389,9 +1389,12 @@ def plot_evoked_topomap(evoked, times="auto", ch_type=None, layout=None,
     vmin = np.min(vlims)
     vmax = np.max(vlims)
     if cmap == 'interactive':
-        cmap = (None, True)
+        cmap = ('Reds' if vmin >= 0. else 'RdBu_r', True)
     elif not isinstance(cmap, tuple):
+        if cmap is None:
+            cmap = 'Reds' if vmin >= 0. else 'RdBu_r'
         cmap = (cmap, False if len(times) > 2 else True)
```
Comment (agramfort) [inline mne/viz/topomap.py]: could we avoid having these hard coded default cmaps in different places in the code?

Comment (wmvanvliet) [inline mne/viz/topomap.py]: we could add an option to the defaults to specify the 'sequential' and 'diverging' color maps.

File: mne/viz/topomap.py [hunk 2]
```diff
@@ -1127,15 +1118,18 @@ def plot_tfr_topomap(tfr, tmin=None, tmax=None, fmin=None, fmax=None,
     im, _ = plot_topomap(data[:, 0], pos, vmin=vmin, vmax=vmax,
                          axes=ax, cmap=cmap[0], image_interp='bilinear',
                          contours=False, names=names, show_names=show_names,
-                         show=False, onselect=selection_callback)
+                         show=False, onselect=selection_callback,
+                         sensors=sensors, res=res, head_pos=head_pos,
+                         outlines=outlines)
```
Comment (agramfort) [inline mne/viz/topomap.py]: is that a bug fix?

Comment (jaeilepp) [inline mne/viz/topomap.py]: Yes. Some of the arguments were not passed.

**Comment by agramfort** [comment]
ok thanks @jaeilepp

**Pull request merged**